kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/OperatorTestUtils.java

```diff
@@ -45,4 +65,27 @@ static boolean isKubeClientAvailable() {
             return client != null;
         }
     }
+
+    public static void preloadOperandImage() {
+        try (var client = kubeClient()) {
+            String operandImage = ProxyDeployment.getOperandImage();
+            var pod = client.run().withName("preload-operand-image")
+                    .withNewRunConfig()
+                    .withImage(operandImage)
+                    .withRestartPolicy("Never")
+                    .withCommand("ls").done();
+            try {
+                client.resource(pod).waitUntilCondition(Readiness::isPodSucceeded, 2, TimeUnit.MINUTES);
```

Comment:
Waiting for readiness will help in case of any transient issues but it does feel a bit sluggish waiting 2 minutes to find out we've been in image backoff the whole time. You could do something like wait until ready || pod has been constantly in (ErrImagePull,ImagePullBackOff) for > N seconds. But it makes it more complicated.